src/simplelatedeliveryandpenalty/sample.txt

```diff
@@ -0,0 +1 @@
+Late Delivery and Penalty. In case of delayed delivery of Goods, "Dan" (the Seller) shall pay to "Steve" (the Buyer) shall pay a penalty amounting to 10.5% of the total value of the Goods for every 2 days of delay. The total amount of penalty shall not, however, exceed 55% of the total value of the delayed goods. If the delay is more than 15 days, the Buyer is entitled to terminate this Contract.
```

Comment:
1. buyer must pay to seller

2. change "Dan" to "Betty Buyer" and "Steve" to "Steve Seller" (in appropriate places) per 1

3. delete "(the Seller)" and "(the Buyer)"